homeassistant/components/matter/select.py

```diff
@@ -21,6 +21,14 @@
 from .helpers import get_matter
 from .models import MatterDiscoverySchema
 
+NUMBER_OF_RINSES_STATE_MAP = {
+    clusters.LaundryWasherControls.Enums.NumberOfRinsesEnum.kNone: "Off",
+    clusters.LaundryWasherControls.Enums.NumberOfRinsesEnum.kNormal: "Normal",
+    clusters.LaundryWasherControls.Enums.NumberOfRinsesEnum.kExtra: "Extra",
+    clusters.LaundryWasherControls.Enums.NumberOfRinsesEnum.kMax: "Max",
```

Comment:
the values here should be lowercase, snake case.
The pretty value can be added in the translations